### DISCUSSION TITLE
Manually trigger indexing via Python script

### DISCUSSION THREAD

**Discussion by Deldion** [photoprism/photoprism#4551] [Q&A]
So,
I am using Nextcloud to upload photos from my devices.
As of now, I have a cronjob that tells Photoprism to index, should there be new images on a specific folder path that changes based on a  `/year/month` structure.
It works fine, but I would like to remove the cronjob that calls the command through the container of Photoprism.
`k3s kubectl exec -n ix-photoprism $(k3s kubectl get pods -n ix-photoprism --field-selector=status.phase=Running -o=jsonpath='{.items[0].metadata.name}') -- photoprism index {path/to/index}`

I would like though to try and do it by making an http request to the Photoprism's REST API.

I am able to get information from the `/api/v1/config` endpoint, which indicates that my access token is valid.
But whenever I try to call the index endpoint, I am getting return code 400, 'this is not possible'.

I've come this far:
```
base_url="https://my_photoprism_instance.net:9003"
    access_token='valid_token_generated_from_UI'
    auth_url = base_url + "/api/v1/index"
    headers = {
            "Authorization": "Bearer " + access_token,
            "Content-Type": "application/json"
        }
    data = {
        "convert":False,
        "path":"/2024/09",
        "rescan":False,
        "skipArchived":True
    }
    response = requests.post(url=auth_url ,headers=headers, data=data)

    print(response)
```

**Comment by srett** [comment]
I'm not fluent in Python, but I'd assume this sends the `data` as url/form-encoded, not json, so there's probably a setting or additional parameter to `.post`.
